### ISSUE TITLE
`Run Script` bug when trying to import cutter

### ISSUE THREAD

**Issue by re-fox** [radareorg/cutter#2271]
**Environment information**
* Operating System: Ubuntu 20.04
* Cutter version: 1.10.3 Appimage
* Obtained from:
  - [ ] Built from source
  - [x] Downloaded from release
  - [ ] Distribution repository
* File format:
Any

**Describe the bug**
I have a script that I want to run within the context of Cutter, however it appears that `import cutter` does not work when running the script from `File -> Run Script`

**To Reproduce**
Start with the small python script
```python
try:
    import cutter
    print("cutter imported")
except:
    print("cutter not found")
```
Save the script and click on `File` -> `Run Script`
![menu](https://user-images.githubusercontent.com/57954766/86160611-ffc52080-bad9-11ea-9847-d00e7596a83f.png)

The following error is produced in the console
```
cutter not found
Cannot open ttyname(0) (null)
```
![console](https://user-images.githubusercontent.com/57954766/86160771-40249e80-bada-11ea-9859-41280f9a3b34.png)

The script that I'm running doesn't need support for QT widgets or a plugin, is there a recommended way to run a script and `import cutter` correctly?

**Expected behavior**
Script will `import cutter`

**Screenshots**
Included above 

**Additional context**

**Comment by re-fox** [comment]
I have downloaded the latest version of Cutter `Version 1.11.0` (AppImage) and verified that this bug still exists.

Is there another recommended way to run a script in cutter?

**Comment by karliss** [comment]
"Run Script" menu is for running r2 scripts. That is files containing list of r2 commands. For example
```
?e hello world
?v 10+5
pdf
```

**Comment by ITAYC0HEN** [comment]
Hey @re-fox, thanks for the follow-up :)
I made a PR (#2330) to make sure it is more clear that this button should be used for executing radare2 scripts and not Cutter python scripts.

Because we dropped the built-in support for Jupyter notebook, we currently suffering from a lack of a proper way to execute Python script file or commands (integrated REPL). We hope to make it better in the coming releases.

Thank you!
